### DISCUSSION TITLE
Rotating backups - auto mode

### DISCUSSION THREAD

**Discussion by jadegouws** [abbbi/virtnbdbackup#86] [Q&A / Support] [answered]
HI Abbbi, Great work so far.

I just want to confirm something about the auto mode, README said it will keep a month.
If I'm using the same folder and automate the backups to run daily will it do incremental backups until it reaches 30 incrementals and then do a full and clean up the incrementals or just keep running?
Basically I'm trying to automate the backups for a weekly cycle, so 1 full and 6 incrementals and rotate again on the 7th day. 

JG

**Comment by abbbi** [comment, marked as answer]
as soon as you switch to an new empty target directory, a full backup will be executed, so how many cycles you do depends on the name of the target directory and how you rotate it (on weekly, daily, or monthly base).

so if you use something like:

`virtnbdbackup .. -o "$(date "+%Y-%m")"`

youll get monthly rotation. Once the month rotates, it will remove all prior checkpoints via libvirt API and starts a 
new incremental backup chain. It will not remove or delete the backups in other directories you have saved.

**Comment by jadegouws** [comment]
Thanks, I generally would only want the most recent backup and maybe a few days before, I'll script something so that it will work on a 7 day rotation and remove the old backups
